Modules/Devices/src/Devices.Application/Users/Commands/SeedTestUsers/SeedTestUsersCommand.cs

```diff
@@ -0,0 +1,6 @@
+﻿using MediatR;
+
+namespace Backbone.Modules.Devices.Application.Users.Commands.SeedTestUsers;
+public class SeedTestUsersCommand : IRequest<SeedTestUsersResponse>
```

Comment:
There is no need to define a Response when it doesn't have any properties.

```suggestion
public class SeedTestUsersCommand : IRequest
```